azure-pipelines.yml

```diff
@@ -210,3 +210,14 @@ jobs:
       testResultsFiles: '**/test-*.xml'
       failTaskOnFailedTests: true
       testRunTitle: 'Publish test results for Python $(PYTHON_VERSION) $(BITS)-bit $(TEST_MODE) Windows'
+
+- job: Linux_PyPy3
+  pool:
+    vmIMage: 'ubuntu-16.04'
+  steps:
+  - script: source tools/pypy-test.sh
+    displayName: 'Run PyPy3 Build / Tests'
+  - task: PublishTestResults@2
+    inputs:
+      testResultsFiles: '**/test-*.xml'
+      testRunTitle: 'Publish test results for PyPy3'
```

Comment:
we should add `failTaskOnFailedTests: true` for consistency with our recent move to displaying test summaries on failures and only failing after the publishing happens

Reply:
+1 to this and the above comment

Reply:
Is there a way to make `failTaskOnFailedTests: true` the default so we don't have to add it to each job?

Reply:
Not sure, but you've fixed the issue so marking as resolved